src/app/views/media-coverage/media-coverage.component.scss

```diff
@@ -1,52 +1,23 @@
-.custom-card{
-  padding: 20px;
-
-  img{
-  width: 300px;
-  margin:5px
-  }
-  margin:10px;
-  .card-text{
-    .col-12{
-      margin:0;
-    }
-
-  }
+.scroll-to-top {
+  position: fixed;
+  bottom: 20px;
+  right: 20px;
+  z-index: 1000;
+  display: none;
+  background-color: #0056b3;
+  color: #fff;
+  border: none;
+  border-radius: 50%;
+  width: 40px;
+  height: 40px;
+  font-size: 20px;
+  cursor: pointer;
+  outline: none;
+  transition: background-color 0.3s ease;
 }
-.card-bg{
-
-  background: linear-gradient(to bottom, rgba(147, 193, 245, 0.833) 0%,rgb(88, 172, 240) 50%,rgb(93, 169, 235) 51%,rgba(125,185,232,1) 100%);
-  box-shadow: 2px 2px 10px black;
-  img{
-    box-shadow: 2px 2px 10px black;
-    border-radius: 10px;
-  }
+.scroll-to-top:hover {
+  background-color: #004080;
 }
-.font-highlight{
-  background: linear-gradient(to left, rgba(30,87,153,1) 0%,rgba(41,137,216,1) 50%,rgba(32,124,202,1) 51%,rgba(125,185,232,1) 100%);
-  // background: -webkit-linear-gradient(#eee, #333);
-  -webkit-background-clip: text;
-  -webkit-text-fill-color: transparent;
-}
-
-hr{
-  background: linear-gradient(to left, rgba(30,87,153,1) 0%,rgba(41,137,216,1) 50%,rgba(32,124,202,1) 51%,rgba(125,185,232,1) 100%);
-}
-div.d-flex{
-  max-width: 100%;
-}
-// *{
-//   overflow-x: hidden;
-// }
-@media(min-width:768px){
-
-  html{
-    -ms-overflow-style: none;  /* IE and Edge */
-    scrollbar-width: none;  /* Firefox */
-  }
-  ::-webkit-scrollbar {
-    display: none;/* Hide scrollbar for Chrome, Safari and Opera */
-    width: 0px;
-    background: transparent; /* make scrollbar transparent */
-  }
+.text{
```

Comment:
rename class to text-justify

Reply:
Not Working...so did it. Now also I have checked but it's not working.